Src/World/World.cpp

```diff
@@ -5,13 +5,17 @@
 #include <Color/Color.h>
 #include <filesystem>
 
+#include "../../Engine/Libraries/SDL2/include/sdl.h"
```

Comment:
SCPCB should not include any SDL headers, only PGE should. This is not permissible. Remove it.

Reply:
Leftover test code, is removed. SDL should only appear in the sound folder at the moment.